app/models/course/video.rb

```diff
@@ -32,6 +34,12 @@ class Course::Video < ApplicationRecord
     end
   end)
 
+  scope :unwatched_by, (lambda do |user|
```

Comment:
@yichenshen can add a test for this scope? Mostly to ensure that the query runs properly works even if rails internal API changes.

Reply:
Alright, sure